### DISCUSSION TITLE
Q: 如何设置多页面共用同一个评论页面？

### DISCUSSION THREAD

**Discussion by parasolwaddledee** [walinejs/waline#1950] [Q&A] [answered]
### Describe the bug | 问题描述

比如两个页面使用同一个评论页面。

### Website URL | 问题网站

none

### Where your waline deploy? | 服务部署在哪里？

Railway.app

### Where your comment data store? | 数据存储在哪里？

PostgreSQL

**Comment by lizheming** [comment, marked as answer]
初始化 Waline 的时候 path 设置成一样即可
